netket/models/deepset.py

```diff
@@ -0,0 +1,104 @@
+from typing import Union, Tuple, Any
+
+import jax
+from jax import numpy as jnp
+from flax import linen as nn
+from netket.utils.types import NNInitFunc
+from jax.nn.initializers import (
+    zeros,
+    ones,
+    lecun_normal,
+)
+
+
+class DeepSet(nn.Module):
+    L: jnp.float64  # boxsize
+    sdim: int  # nbr of spatial dimensions
+
+    """Number of layers in phi/rho network"""
```

Comment:
The doc strings should be one per attribute, in the line right below (not above)